config/graphene_wpe_config.toml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[KvStorage]
+# the remote version is of higher priority if enabled
+remote_url = "http://localhost:9090"
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# ------------------------------------------------------------------
+# Work load execution-settings for workload execution(synchronous/asynchronous)
+# ------------------------------------------------------------------
+[WorkloadExecution]
+# Set this flag to 1 for sync workorder execution.Else set it to 0
+sync_workload_execution = 0
+
+# -------------------------------------------------------------------
+# EnclaveManager -- configuration of Intel SGX Enclave EnclaveManager
+# -------------------------------------------------------------------
+[EnclaveManager]
+# configurations for asynchronous work load execution
+sleep_interval = "2"
+# configurations for synchronous Work load execution
+# zmq_port is the port used for zmq socket communication
+# Port for socket interaction with listener
+zmq_port = '5555'
+# Graphene worker url. Graphene worker uses ZMQ for communication
+graphene_zmq_url = "tcp://graphene-python-worker:7777"
```

Comment:
Please change zmq_url to "tcp://graphene-python-workload:7777" because service name is changed to graphene-python-workload.

Reply:
done